### PR TITLE
Add tests to parseStrToNumStrict

### PR DESCRIPTION
**Pull request by banciur**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 coverage
 yarn-error.log
 webpack-stats.json
+setupTests.js
```
Comment (krzkaczor) [inline .gitignore]: why do you want to ignore it?

File: app/typings/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.js
```
Comment (krzkaczor) [inline app/typings/.gitignore]: why is this needed? there should be no js files simply.

File: test/utils/utils.spec.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/* tslint:disable: only-arrow-functions ter-prefer-arrow-callback */
+
+import * as chai from "chai";
+import { parseStrToNumStrict } from "../../app/utils/utils";
+
+describe("utils/utils.ts - parseStrToNumStrict", () => {
+  it("should return Nan for null", function() {
+    const parsed = parseStrToNumStrict(null);
+    chai.assert.isNaN(parsed);
```
Comment (krzkaczor) [inline test/utils/utils.spec.ts]: I would really prefer to stick to `expect(something).to.be.something` API. We had the same discussion on smart contract project.

File: test/utils/utils.spec.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/* tslint:disable: only-arrow-functions ter-prefer-arrow-callback */
+
+import * as chai from "chai";
+import { parseStrToNumStrict } from "../../app/utils/utils";
+
+describe("utils/utils.ts - parseStrToNumStrict", () => {
```
Comment (krzkaczor) [inline test/utils/utils.spec.ts]: you can do (nested describes)
```javascript
describe("utils", () => {
  describe("parseStrToNumStrict", () => {

  });
})
```

File: test/utils/utils.spec.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  });
+
+  it("should return number", function() {
+    const parsed = parseStrToNumStrict("0");
+    chai.assert.isFinite(parsed);
+  });
+
+  it("should handle negative numbers", function() {
+    const parsed = parseStrToNumStrict("-1");
+    chai.assert.isBelow(parsed, 0);
```
Comment (krzkaczor) [inline test/utils/utils.spec.ts]: why don't you assert exact numbers in these tests? like `expect(parsed).to.be.eq(-1)`

File: app/utils/utils.ts [hunk 6]
```diff
@@ -6,6 +6,10 @@ const downloadjs = require("downloadjs") as any;
  * White spaces are removed, "," replaced by "." is there is more than 1 dot or other chars NaN is returned.
  */
 export const parseStrToNumStrict = (source: string): number => {
+  if (source === null) {
```
Comment (krzkaczor) [inline app/utils/utils.ts]: Is this possible that this will break forms already using this method? For example neumark estimation will be calculated for NaN for empty input field

Comment (banciur) [reply app/utils/utils.ts]: No if we would pass null value function would get null pointer as [few lines](https://github.com/Neufund/commit.neufund.org/blob/tomek/test_util_parseStrToNumStrict/app/utils/utils.ts#L17) below we call `source.replace`

File: test/utils/utils.spec.ts [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    it("should handle dot and coma", function() {
+      expect(parseStrToNumStrict("1.5")).to.eq(1.5);
+      expect(parseStrToNumStrict("1,5")).to.eq(1.5);
+    });
+
+    it("should handle white space", function() {
+      expect(parseStrToNumStrict("1 5")).to.eq(15);
+    });
+
+    it("do not allow for double dot / coma", function() {
```
Comment (krzkaczor) [inline test/utils/utils.spec.ts]: typo: comma
